frontend/src/app/person-network-members/person-network-members.component.ts

```diff
@@ -0,0 +1,74 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { NetworkMemberModel } from '../models/network-member.model';
+import { ConfirmService } from '../confirm.service';
+import { NetworkMemberService } from '../network-member.service';
+import { switchMap, tap } from 'rxjs/operators';
+import { PersonModel } from '../models/person.model';
+import { NETWORK_MEMBER_TYPE_TRANSLATIONS } from '../display-network-member-type.pipe';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { NetworkMemberCommand } from '../models/network-member.command';
+import { Observable } from 'rxjs';
```

Comment:
nit: I personally don't like to have imports mixed up. The recommandation is to have angular imports, followed by 3rd party imports, followed by a blank line, and then the imports from our codebase.

Reply:
I there some IntelliJ option to do that? I never write imports or arrange them. I just let the IDE add them and optimize them.

Reply:
Sadly, I think there is not (AFAIK). Maybe we could find a TSLint rule for that. I usually reorder by hand when I'm done...